kousa/lib/broth/message/misc/search.ex

```diff
@@ -31,17 +33,24 @@ defmodule Broth.Message.Misc.Search do
       # currently not enforced, but once we have real DisplayRoom and
       # DisplayUser schemas we'll make sure Search.search outputs those.
       field(:items, {:array, :map})
+      field(:rooms, {:array, :map})
+      field(:users, {:array, :map})
```

Comment:
This should be like https://github.com/benawad/dogehouse/blob/staging/kousa/lib/broth/message/user/get_followers.ex#L27 for the rooms/users

Reply:
done